src/layouts/govuk.scss

```diff
@@ -0,0 +1 @@
+@import '@govuk-fronend/all';
```

Comment:
typo

Reply:
thanks ... this file is redundant ... will remove

Reply:
Thanks for pointing that out! We scratched our heads why did webpack not error out and discovered this file is no longer used.